packages/edit-site/src/components/global-styles/font-library-modal/utils/index.js

```diff
@@ -135,39 +136,84 @@ export function getDisplaySrcFromFontFace( input, urlPrefix ) {
 	return src;
 }
 
-export function makeFormDataFromFontFamily( fontFamily ) {
+export function makeFontFamilyFormData( fontFamily ) {
 	const formData = new FormData();
 	const { kebabCase } = unlock( componentsPrivateApis );
 
-	const newFontFamily = {
-		...fontFamily,
+	const { fontFace, category, ...familyWithValidParameters } = fontFamily;
```

Comment:
are there families with invalid parameters?

Reply:
Invalid parameters according to the REST schema and data validation, it returns an error when `fontFace` or `category` are present in data.